Sources/WASI/WASI.swift

```diff
@@ -803,23 +815,19 @@ extension WASI {
 
         ]
 
-        var preview1: [String: HostFunction] = unimplementedFunctionTypes.reduce(into: [:]) { functions, entry in
+        var preview1: [String: WASIHostFunction] = unimplementedFunctionTypes.reduce(into: [:]) { functions, entry in
             let (name, type) = entry
-            functions[name] = HostFunction(type: type) { _, _ in
+            functions[name] = WASIHostFunction(type: type) { _, _ in
                 print("\"\(name)\" not implemented yet")
                 return [.i32(WASIAbi.Errno.ENOSYS.rawValue)]
             }
         }
 
         func withMemoryBuffer<T>(
```

Comment:
I should note that this function is basically now a no-op — I wanted to minimize the diff to begin with for ease of review but it might make sense to remove this now. @kateinoigakukun would you prefer that I do it in this PR or in a follow-up?

Reply:
Thanks for minimizing diff! I prefer a following up way